models/review.js

```diff
@@ -32,9 +40,34 @@ module.exports = (sequelize, DataTypes) => {
     },
     source: {
       type: DataTypes.STRING,
-      defaultValue: 'Streaming',
-    }
-  });
+      defaultValue: 'Streaming'
+
+  }
+},
+
+
 
-  return Post;
+{timestamps: false});
+
+
+  return Review;
 };
+
+
+  
+    // id: {
```

Comment:
Make sure you delete any unnecessary comments when creating a PR, this stuff shouldn't be making it into the main code

Reply:
Okay can do - again do I do this now or afte the PR?